frontend/styles/loginStyles.ts

```diff
@@ -0,0 +1,50 @@
+import { Theme } from "@mui/material/styles"
+
+const loginStyles = (theme: Theme): { [key: string]: React.CSSProperties } => ({
+  loginContainer: {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    height: "100vh",
+  },
+  loginGrid: {
+    maxWidth: 545,
+    p: 5,
+  },
```

Comment:
Can we move this file in the folder where we have the login page? Let's keep styles of a page with the component. 
Also file names should not be camelCase. Rename `themeProvider.tsx` to `theme-provider.tsx`